content/en/docs/concepts/architecture/nodes.md

```diff
@@ -27,14 +27,14 @@ The [components](/docs/concepts/overview/components/#node-components) on a node
 
 ## Management
 
-There are two main ways to have Nodes added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
+There are two main ways to have nodes added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
```

Comment:
When I read this sentence, it seems as if the nodes are physically added to the API server.
Instead:

There are two ways to add nodes to the API server:
The kubelet running on a node registers with the control plane
A user manually adds a `Node` using [add how this is added]

Remove "object" after resource name:
After you create a `Node` or the kubelet on the node self-registers, ...
the new `Node` is valid.

Reply:
OK.

Reply:
How about:
```suggestion
Before the control plane can [schedule](/docs/concepts/scheduling-eviction/) any pods to run on
a node, the node needs to be registered with the cluster. There are two common ways to
register nodes with the control plane:
1.  The kubelet, running on a node, [registers itself](#self-registration-of-nodes) with
    the cluster's
    {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}.
1.  A cluster administrator [manually](#manual-node-administration) adds a `Node` using a tool such as `kubectl`.
```

Now ideally https://k8s.io/docs/concepts/scheduling-eviction/ should include the glossary definition for “scheduling”, but I'd rather link there now and let the smart reader click through to the first page in that section if they wanted to learn more.

Reply:
@bergerhoffer when you're adding it to the API server, it's a Node (an object with `kind: Node`).